### PR TITLE
don't reload angular if it is already loaded

### PR DESCRIPTION
**Pull request by DrHofman**
This removes the WARNING: Tried to load angular more than once. message...

### CODE REVIEW

File: angular-chart.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
   'use strict';
   if (typeof exports === 'object') {
     // Node/CommonJS
-    module.exports = factory(require('angular'), require('Chart.js'));
+    module.exports = factory(angular ? angular : require('angular'), require('Chart.js'));
```
Comment (jtblin) [inline angular-chart.js]: That would create an error if angular is not defined. Use `window.angular` or `typeof angular`.

**Comment by tommasosperoni** [comment]
+1

**Comment by jtblin** [comment]
@DrHofman thanks for the contribution but as per the comment that would fail if angular is not defined. Do you plan to do the change and resolve the conflict?

File: angular-chart.js (hunk 1 above)
Comment (DrHofman) [inline angular-chart.js]: you are correct I will update this

**Comment by DrHofman** [comment]
updated

**Comment by jtblin** [comment]
Thank you @DrHofman :+1:

**Comment by tommasosperoni** [comment]
thanks!

**Pull request merged**